apps/dashboard/src/components/contract-components/tables/cells.tsx

```diff
@@ -75,11 +76,32 @@ export const AsyncContractTypeCell = memo(
     const publishedContractsFromDeployQuery =
       usePublishedContractsFromDeploy(contract);
 
+    const publishedContractsFromDeployOriginal =
+      publishedContractsFromDeployQuery.data || [];
+
+    const publishedContractsFromDeploySorted = [
+      ...publishedContractsFromDeployOriginal,
+    ]
+      // latest first
```

Comment:
this is what confused me @jnsdls - is latest first or earliest first?

Reply:
@MananTank gotta make sure we don't double-reverse

we want the *latest* version that matches to be shown

Reply:
not double reversing there - the last one (latest) is put first + then we sort while maintaining this order and put the thirdweb publisher first -> so we get thirdweb publisher with latest version as first element